SU-SWS/decanter
# Linting Cleanup

# READY FOR REVIEW

# Summary
- Coding standards update.

# Needed By (Date)
- Whenever

# Urgency
- Low

# Steps to Test

1. Nothing should have changed so please review the formatting.
2. Run `npm run eslint` and review output
3. Run `npm run sasslint` and review output
